library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -31,8 +33,14 @@ void addCallback(MigrationCallback callback) {
     }
 
     public void run() {
-        InternalMigrationStatus migrationStatus = new VersionOneToVersionTwoMigrationStatus(MigrationStatus.Status.DB_NOT_PRESENT);
         if (!databasePath.exists()) {
+            InternalMigrationStatus migrationStatus = new VersionOneToVersionTwoMigrationStatus(
+                    jobIdentifier,
+                    MigrationStatus.Status.DB_NOT_PRESENT,
+                    0,
```

Comment:
can we extract those numbers as constants with names?

Reply:
64ad585